library/src/modules/app/open-api-module.js

```diff
@@ -2,10 +2,58 @@
 import Vue from 'vue'
 import axios from 'axios'
 
+
+
+
 let _initCallbackFn = null
 let _initialized = false
 let _token = null
 let _baseURL = null
+let _refreshToken = null
+
+const httpModule = (config, {baseURL, token, refreshToken, clientId}) => {
+  const http = axios.create(config)
```

Comment:
I like what the restructuring does for code readability here. I haven't used it like that before.